IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeTelecommand/DocPropertySet.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CommunicationsApplianceTypeTelecommand" UniqueId="0802509f-6afc-2723-77df-3bc3ab6e7742" ApplicableType="IfcCommunicationsAppliance/TELECOMMAND" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NumberOfDiagnosticMaintenanceWorkstations_1NteTJwCTDPxZ71U8R760" />
+		<DocProperty xsi:nil="true" href="NumberOfCPUs_1rjaY2H_LBUB1VPn5lebft" />
+		<DocProperty xsi:nil="true" href="NumberOfTrafficElectricalTractionWorkstations_2lLmk3LIf6ew_EjtIAMdjz" />
+		<DocProperty xsi:nil="true" href="PresenceOfDiagnostic_1s3CI0yTT8hAw1ZrWl4DYo" />
```

Comment:
Presence of Diagnostic also seems like an instance property.

Reply:
It is more like an instance property. Again, don't feel good to create one Pset just for this